src/message/mailbox/types.rs

```diff
@@ -108,40 +110,18 @@ impl<S: Into<String>, T: Into<String>> TryFrom<(S, T)> for Mailbox {
     }
 }
 
-/*
-impl<S: AsRef<&str>, T: AsRef<&str>> TryFrom<(S, T)> for Mailbox {
-    type Error = AddressError;
-
-    fn try_from(header: (S, T)) -> Result<Self, Self::Error> {
-        let (name, address) = header;
-        Ok(Mailbox::new(Some(name.as_ref()), address.as_ref().parse()?))
-    }
-}*/
```

Comment:
This too is a breaking change

Reply:
Not sure to understand, it is already commented on `master`, I just removed the code. Would you like me to put it back?